pkg/adapter/mtping/runner.go

```diff
@@ -99,6 +100,9 @@ func (a *cronJobsRunner) AddSchedule(cfg PingConfig) cron.EntryID {
 		ResourceGroup: resourceGroup,
 	}
 	ctx = kncloudevents.ContextWithMetricTag(ctx, metricTag)
+	ctx = logging.ContextWithMTLoggerFromLogger(ctx, a.Logger, cfg.Namespace)
+	logger := logging.MTLoggerFromContext(ctx)
+	logger.TenantInfof("Created pingsource %s with schedule %s", cfg.Name, cfg.Schedule)
```

Comment:
hmm this is a bit strange. There is no need to extend the context this it's being dropped. I would expect a.Logger to be a MTLogger and just do:
```
a.Logger.TenantInfof(cfg.Namespace, ...)
```

WDYT?

Reply:
That would be nice but there isn't a unique CronJobsRunner per client/namespace is there? If no, how to distinguish which namespace/client?

Reply:
that's right. That's why I suggest to change the `TenantXX` functions to take the namespace as first argument.

Reply:
I haven't cracked the code, but I'm curious how you expect this information to manifest and how it's different from things like the "key", which we already decorate many loggers with (enabling filtering by both namespace and resource)?